### ISSUE TITLE
xerox clipboard copy crash mpsyt

### ISSUE THREAD

**Issue by rcspam** [mps-youtube/mps-youtube#311]
Here what i am after type "x 1"
```
...
20   Basic mixing in Ardour 3: Part 5 - Basic mix, groupin  Tux van Beetho  16/08/14  5     5.00  8     0     People &   1152  34:22
21   Manual Ardour 2 - Lo básico de la Edición de Audio     ManualAnalfate  24/11/11  9     4.85  34    1     Science &  6976  11:32
22   Teebs - Wind Loop                                      Alpha Pup       25/01/11  26    4.98  334   1     Music      48k   03:15

Search results for ardour                                                                                      [1/23>
> x 1
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 4119, in clip_copy
    xerox.copy(link)
AttributeError: 'module' object has no attribute 'copy'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/mpsyt", line 9, in <module>
    load_entry_point('mps-youtube==0.2.5', 'console_scripts', 'mpsyt')()
  File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 4780, in main
    if matchfunction(k, v, userinput):
  File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 4680, in matchfunction
    func(*matches)
  File "/usr/local/lib/python3.4/dist-packages/mps_youtube/main.py", line 4123, in clip_copy
    except xerox.base.ToolNotFound as e:
AttributeError: 'module' object has no attribute 'base'
```

**Comment by lol768** [comment]
I can reproduce.

**Comment by ids1024** [comment]
I have already noticed this, but forgot since I have no use for the feature. Apparently I left a comment in January on the xerox issue @lol768 linked to. I think I'll change this to use pyperclip, since that seems to be maintained.

**Comment by ids1024** [comment]
Fixed in commit 0e3a9ca0b9cfa575b8a5d0030eb1bfe9d661c67c.
